ia/src/protocole.py

```
@@ -0,0 +1,128 @@
+
+def handle_fork(client, response) -> None:
+    server_address = client["socket"].getpeername()
+    team_name = client["team_name"]
+    connect_client(server_address, team_name)
+
+def handle_Broadcast(client, response) -> None:
+    if (response == "ok"):
+        return
+    print(f"Message broadcast reçu: {response}")
+
+def handle_commande(client, commande, response):
```

Ruff (0.11.9)

Missing return type annotation for public function `handle_commande`

Add return type annotation: `None`

(ANN201)